react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkCheckboxItem = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    checkboxAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    isDisabled,
+    onChangeFunc,
```

Comment:
We recently changed this in other inputs to be onChange instead. I messaged @RVMendoza  about updating this one. Leaving a comment to remember!

Reply:
fixed